metadata/src/main/resources/common/metadata/UnregisterBrokerRecord.json

```diff
@@ -19,9 +19,9 @@
   "name": "UnregisterBrokerRecord",
   "validVersions": "0",
   "fields": [
-    { "name": "BrokerId", "type": "int32", "versions": "0+",
```

Comment:
Previously, we agreed to use BrokerId here for consistency with the other records.